framework/src/Volo.Abp.BlazoriseUI/Components/PageHeader.razor

```diff
@@ -0,0 +1,35 @@
+﻿<Row Class="entry-row">
+    <Column ColumnSize="ColumnSize.IsAuto">
+        <h1 class="content-header-title">@Title</h1>
+    </Column>
+    @if (BreadcrumbItems.Any())
+    {
+        <Column ColumnSize="ColumnSize.IsAuto.OnWidescreen" Class="pl-lg-0">
+            <Breadcrumb Mode="@(BreadcrumbShowCurrent ? BreadcrumbMode.Auto : BreadcrumbMode.None)">
+                @if (BreadcrumbShowHome)
+                {
+                    <BreadcrumbItem>
+                        <BreadcrumbLink To="/"><i class="fa fa-home"></i></BreadcrumbLink>
```

Comment:
Blazorise has it's own Icon component, so instead of `<i class="fa fa-home"></i>` write `<Icon Name="IconName.Home" />`

Reply:
Thanks. I will move the BreadcrumbItem class to the BlazoriseUI assembly. So that developers can use Blazorise IconName enum.